File: AdvancedScada.Controls/HslControls/Keypad/KeyPadEventArgs.cs
using System;

namespace AdvancedScada.Controls.HslControls.Keypad
{
    public class KeypadEventArgs : EventArgs
    {
        private string m_Key;

        public string Key
        {
            get
            {
                return this.m_Key;
            }
        }

        public KeypadEventArgs(string Key)
        {
            this.m_Key = Key;
        }
    }


}

File: HslScada.Controls_Net45/Keyboard/KeypadEventArgs.vb
Imports System

Public Class KeypadEventArgs
    Inherits EventArgs
    Private m_Key As String

    Public ReadOnly Property Key As String
        Get
            Return Me.m_Key
        End Get
    End Property

    Public Sub New(ByVal Key As String)
        MyBase.New()
        Me.m_Key = Key
    End Sub
End Class
